imports/blocks/checkout-buttons/index.js

```diff
@@ -188,8 +195,4 @@ class CheckoutButtons extends Component {
   }
 }
 
-export default withSavedPayments(withState(CheckoutButtons));
-
-export {
-  CheckoutButtons as CheckoutButtonsWithoutData,
-};
```

Comment:
@jbaxleyiii if we don't do this here, then we run into the problem that we were seeing with the linter not liking the import of the component in tests (SubFund was the example you and I talked about the other day) and then we have to disable linting on that line. We can't have two exports named the same thing, from a linting point of view. Is there a different way to solve the linting problem?